<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QMainWindow" name="AdminPanel">
    <property name="geometry">
     <rect><x>0</x><y>0</y><width>1000</width><height>700</height></rect>
    </property>
    <property name="windowTitle">
     <string>Panel de Administración - MenULE</string>
    </property>
    <property name="windowState">
     <set>Qt::WindowFullScreen</set>
    </property>
    <property name="styleSheet">
     <string notr="true">
QMainWindow {
        background-color: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                                                            stop:0 #2e7d32, stop:1 #a5d6a7);
}
QPushButton {
        background-color: #388e3c;
        color: white;
        font-weight: bold;
        font-size: 18px;
        border-radius: 10px;
        padding: 12px 24px;
        max-width: 300px;
        min-width: 200px;
        margin-left: auto;
        margin-right: auto;
}
QPushButton:hover {
        background-color: #2e7d32;
}
     </string>
    </property>
    <widget class="QWidget" name="centralwidget">
     <layout class="QHBoxLayout" name="mainLayout">
        <item>
         <widget class="QTabWidget" name="tabPanel">
            <property name="tabPosition">
             <enum>QTabWidget::North</enum>
            </property>

            <widget class="QWidget" name="tabUsuarios">
             <attribute name="title">
                <string>Usuarios</string>
             </attribute>
             <layout class="QVBoxLayout" name="layoutUsuarios">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnAgregarUsuario">
                    <property name="text">
                     <string>Agregar Usuario</string>
                    </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnEliminarUsuario">
                    <property name="text">
                     <string>Eliminar Usuario</string>
                    </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnDarDeBaja">
                    <property name="text">
                     <string>Dar de Baja</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTableWidget" name="tablaUsuarios"/>
                </item>
             </layout>
            </widget>

            <widget class="QWidget" name="tabMenus">
             <attribute name="title">
                <string>Menús</string>
             </attribute>
             <layout class="QVBoxLayout" name="layoutMenus">
                <item>
                 <spacer name="spacerTop">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size><width>20</width><height>100</height></size>
                    </property>
                 </spacer>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnModificarMenu">
                    <property name="text">
                     <string>Modificar Menú</string>
                    </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnVerMenuAdmin">
                    <property name="text">
                     <string>Ver Menú</string>
                    </property>
                 </widget>
                </item>
                <item>
                 <spacer name="spacerBottom">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size><width>20</width><height>100</height></size>
                    </property>
                 </spacer>
                </item>
             </layout>
            </widget>

            <widget class="QWidget" name="tabReservas">
             <attribute name="title">
                <string>Reservas</string>
             </attribute>
             <layout class="QVBoxLayout" name="layoutReservas">
                <item>
                 <widget class="QTableWidget" name="tablaReservas"/>
                </item>
             </layout>
            </widget>

            <widget class="QWidget" name="tabPagos">
             <attribute name="title">
                <string>Pagos</string>
             </attribute>
             <layout class="QVBoxLayout" name="layoutPagos">
                <item>
                 <widget class="QTableWidget" name="tablaPagos"/>
                </item>
             </layout>
            </widget>

            <widget class="QWidget" name="tabConfiguracion">
             <attribute name="title">
                <string>Configuración</string>
             </attribute>
             <layout class="QVBoxLayout" name="layoutConfiguracion">
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnCambiarContrasena">
                    <property name="text">
                     <string>Cambiar Contraseña</string>
                    </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                 <widget class="QPushButton" name="btnCerrarSesion">
                    <property name="text">
                     <string>Cerrar Sesión</string>
                    </property>
                 </widget>
                </item>
                <item alignment="Qt::AlignHCenter">
                    <widget class="QPushButton" name="btnDescargarBD">
                     <property name="text"><string>Descargar base de datos</string></property>
                    </widget>
                 </item>
             </layout>
            </widget>

         </widget>
        </item>
     </layout>
    </widget>
    <widget class="QMenuBar" name="menubar"/>
    <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
